### PR TITLE
#234  Fix backup permissions problem. stb2

### LINKED ISSUE #234: se-backup-restore.sh:6-9: - DB restore error on local env. How...
Required for #190 

-----

The puzzle `232-865d5cdb` (from #232) in [`etc/se-backup-restore.sh`](https://github.com/fidals/shopelectro/blob/master/etc/se-backup-restore.sh) (lines 6-9) has to be resolved: "- DB restore error on local env. How to reproduce: - launch this script - restart se-python container"

The puzzle was created by duker33 on 28-Nov-17. 

Estimate: 0 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by duker33**
Closes #234

### CODE REVIEW

File: docker/cron/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,8 @@
 FROM debian:latest
 
-RUN apt-get update && apt-get install -y \
-    cron \
-    rsyslog
+RUN apt-get update && apt-get install -y cron
```
Comment (artemiy312) [inline docker/cron/Dockerfile]: rsyslog writes log messages, these help us to check health of cron's jobs.
e.g. ( I did not mount /opt/backup/* files, so an  occurred error, that was logged)  
```bash
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 liblogging-stdlog:  [origin software="rsyslogd" swVersion="8.24.0" x-pid="13" x-info="http://www.rsyslog.com"] start
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 cron[23]: (CRON) INFO (pidfile fd = 3)
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 cron[24]: (CRON) STARTUP (fork ok)
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 cron[24]: (CRON) INFO (Skipping @reboot jobs -- not system startup)
se-backup-data | Jul  8 23:03:36 3acb1ea39ba1 liblogging-stdlog: imklog: cannot open kernel log (/proc/kmsg): Operation not permitted.
se-backup-data | Jul  8 23:03:36 3acb1ea39ba1 liblogging-stdlog: activation of module imklog failed [v8.24.0 try http://www.rsyslog.com/e/2145 ]
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 CRON[27]: (root) CMD (       sh /usr/bin/entrypoint.sh 2>&1 | logger)
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: database
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: /usr/app/src/database: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: media
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): /opt/backup/database-2018-07-08.tar.gz: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: /usr/app/src/media: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: static
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): /opt/backup/media-2018-07-08.tar.gz: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: /usr/app/src/static: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): /opt/backup/static-2018-07-08.tar.gz: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: find: '/opt/backup/*': No such file or directory
```

Why did you remove a log processing from this Dockerfile? How does it help us to resolve a permissions problem?

**Comment by artemiy312** [comment]
@duker33 , Can you provide an error message to clarify the PR?

**Comment by duker33** [comment]
hmm, seems i can't reproduce it. Close
